pyof/foundation/network_types.py

```diff
@@ -50,35 +49,30 @@ def get_hash(self):
         return hash(self.pack())
 
 
-class GenericTLV:
+class GenericTLV(GenericStruct):
     """TLV structure of LLDP packets.
 
     This is a Type, Length and Value (TLV) struct.
 
-    The LLDP/TLV definition states that the Type field have 7 bits, while the
-    length have 9 bits. The Value must be between 0-511 octets.
+    The LLDP/TLV definition states that the Type field have 7 bits, while
+    the length have 9 bits. The Value must be between 0-511 octets.
 
-    Internally, on the instances of this class, the Type is a integer (0-127)
-    and the Length is dynamically calculated based on the current type and
-    value.
+    Internally, on the instances of this class, the Type is a integer
+    (0-127) and the Length is dynamically calculated based on the current
+    type and value.
     """
 
     def __init__(self, tlv_type=127, value=BinaryData()):
         """Create an instance and set its attributes."""
-        #: type (int): The Type of the TLV Structure
+        super().__init__()
         self.type = tlv_type
-        #: value (BinaryData): The value of the TLV Structure
-        self._value = value
-
-    @property
```

Comment:
@macartur
1) why have you removed the `value` property?
2) why renaming `_value` to `value`?

I know that we have a goal of removing the "`value`" pointer to access our objects, but I think that we need to keep our library with a common ground. Or we use the `value` or we do not use it, across all source.
Reasonable?